### ISSUE TITLE
Is my result of the mean absolute error reliable?

### ISSUE THREAD

**Issue by njusq** [adobe-research/deep-acoustic-analysis#4]
Hi! Thank you for your complementation.
We want to use data augmentation to improve the performance on ASR especially in reverberate environment. We referenced the paper _Scene-aware Far-field Automatic Speech Recognition_ and found this project.

I got the result from the given code and .h5 data without any modification, the training stage finished at the 50th epoch:
`train_mean_absolute_error:0.2
val_mean_absolute_error:0.24
test_mean_absolute_error: 0.30`

The first utterance in test data shows that:
`Inputfile,t60_truth_0,t60_truth_1,t60_truth_2,t60_truth_3,t60_truth_4,t60_truth_5,t60_truth_6,t60_truth_7,t60_prediction_0,t60_prediction_1,t60_prediction_2,t60_prediction_3,t60_prediction_4,t60_prediction_5,t60_prediction_6,t60_prediction_7,mae
f1_script1_clean_1.wav,0.045012277,0.055239435,0.06365131,0.082025155,0.06356307,0.03540763,0.020986253,0.017730715,1.0521849,0.7734909,0.493748,0.36686748,0.2314288,0.2228179,0.33583656,0.4605934,0.444169`

There are great difference between the ground truth result and prediction from my perspective, because . Is this result reliable？Can I achieve the improvement on ASR using T60 with mean absolute error 0.30?

**Comment by Graphi07** [comment]
Thank you for your interest in our work. The result seems normal. But whether the error is low or high depends on the application. This paper of ours does not heavily rely on a highly accurate T60. Also note that our training and test set are obviously mismatched, for the reason stated in our paper. While I cannot comment on whether this can help your ASR task, I can say the prediction can definitely be improved by 1) using training data that is more suited for your application; and 2) leveraging more sophisticated network architectures such as ResNets.

**Comment by njusq** [comment]
I still have some questions about the ground truth T60 values.  Do you get the ground truth T60 values as the paper _The ACE Challenge - Corpus Description and Performance Evaluation_ described? We tried to get those values using the sub-band AIR envelopes and the tool from http://legacy.spa.aalto.fi/software/decay/ . The result still seems weird. Does the software in ACE Challenge provide the specific process to generate ground truth T60s from AIR?

**Comment by Graphi07** [comment]
We calculated our T60 values using in-house tools, not from the ACE challenge, and I personally don't think they have released any tool for generating the labels. That being said, it shouldn't be a problem if you use a different labeler. You just need to regenerate the labels for both your training and test data without using any pre-trained models, then your results should be consistent.

**Comment by njusq** [comment]
Thank you! We have achieved improvement on our ASR system with your method!
